pkg/apis/config/v1beta1/defaults_test.go

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"github.com/google/go-cmp/cmp"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	schedulerconfig "k8s.io/kube-scheduler/config/v1"
+	"k8s.io/utils/pointer"
+	"strconv"
+	"testing"
+)
+
+func TestSetDefaultsCoschedulingArgs(t *testing.T) {
+	tests := []struct {
+		name   string
+		config *CoschedulingArgs
+		expect *CoschedulingArgs
```

Comment:
Can you merge the top-level tests into one test as sub-tests? Ref: https://github.com/kubernetes/kubernetes/blob/66e1d27a59208f3f90016e0790a13e2602fecdeb/pkg/scheduler/apis/config/v1beta2/defaults_test.go#L456

Reply:
Good idea~ Thanks for your comment ~~ I will update soon~

Reply:
Fix done~